.github/workflows/vitest.yml

```
@@ -0,0 +1,19 @@
+name: 'Tests: vitest'
+
+on:
+  push:
+    branches: [ "main", "ragezbla/89-vitest-gh-actions" ] # TODO remove after checking it works
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '45 15 * * 4'
+
+jobs:
+  cypress-run:
```

This should be related to vitest not cypress
```suggestion
  run-pinia-tests:
```